day1OfJS/datatypes.js

```diff
@@ -0,0 +1,10 @@
+/*
+Create datatypes.js file and use the JavaScript typeof operator to check different data types. Check the data type of each variable
+
+console.log(typeof "Luisa");
+console.log(typeof 5);
+console.log(typeof true);
+console.log(typeof null);
+console.log(typeof undefined);
+
+*/
```

Comment:
Please add a newline line at the end of the file.

Reply:
Lines added - 4f7f6f6